### PR TITLE
Support Firefox

### PR DESCRIPTION
**Pull request by grouma**
Note this changed passed an internal global presubmit.

cc @liuming0

### CODE REVIEW

File: lib/server/sse_handler.dart [hunk 1]
```diff
@@ -17,7 +17,7 @@ String _sseHeaders(String origin) => 'HTTP/1.1 200 OK\r\n'
     'Connection: keep-alive\r\n'
     'Access-Control-Allow-Credentials: true\r\n'
     'Access-Control-Allow-Origin: $origin\r\n'
```
Comment (natebosch) [inline lib/server/sse_handler.dart]: should these be consistent?

File: lib/server/sse_handler.dart [hunk 2]
```diff
@@ -140,7 +140,12 @@ class SseHandler {
     }
     return shelf.Response.ok('', headers: {
       'access-control-allow-credentials': 'true',
-      'access-control-allow-origin': req.headers['origin']
+      'access-control-allow-origin': _getOrigin(req),
     });
   }
+
+  String _getOrigin(shelf.Request req) =>
```
Comment (natebosch) [inline lib/server/sse_handler.dart]: https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

File: lib/server/sse_handler.dart (hunk 1 above)
Comment (grouma) [reply lib/server/sse_handler.dart]: cc @liuming0, what's the impact of changing these all to `\n\n`?

Comment (grouma) [reply lib/server/sse_handler.dart]: Note that making them consistent does not impact local tests.

**Pull request merged**